test/functional/bundleadd/add-json.bats

```diff
@@ -7,7 +7,6 @@ load "../testlib"
 
 test_setup() {
 
-	skip "This test is unstable and should be improved. Skipping that for now"
```

Comment:
In fact all the messages that are non deterministic have been stripped out of the tests. All the messages that are related to steps like "download_packs" or "download_fullfiles" are only looking for the message where the step has reached "100%" of completion, which should always be there. Steps that are still showing other percentages in the tests like "install_files" are deterministic because  they are based on the amount of files and not based on downloaded data so they always have the same percentages.

Reply:
oh, I see that now. you are right. We are good to go